### PR TITLE
Add `ticloud connect` doc

### PR DESCRIPTION
**Pull request by xuanyu66**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/get-started-with-cli.md [hunk 1]
```diff
@@ -125,6 +125,14 @@ To create a Serverless Tier cluster, enter the following command, and then follo
 ticloud cluster create
 ```
 
+### Connect to the cluster
+
+After the cluster is created, you can connect to the cluster using the following command:
```
Comment (qiancai) [inline tidb-cloud/get-started-with-cli.md:130]:
```suggestion
After the cluster is created, you can connect to the cluster:
```

File: tidb-cloud/ticloud-connect.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+Connect to a TiDB Cloud cluster:
+
+```shell
+ticloud connect [flags]
+```
+
+> **Note:**
+>
+> The connection forces the ANSI SQL mode for the session.
+You can get more information about the ANSI SQL mode from [ANSI SQL mode](https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html#sqlmode_ansi)
```
Comment (qiancai) [inline tidb-cloud/ticloud-connect.md:17]:
```suggestion
> The connection forces the [ANSI SQL mode](https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html#sqlmode_ansi) mode for the session.
```

File: tidb-cloud/ticloud-connect.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+## Examples
+
+Connect to a TiDB Cloud cluster in interactive mode:
+
+```shell
+ticloud connect
+```
+
+Use the default user to connect to a TiDB Cloud cluster in non-interactive mode:
```
Comment (qiancai) [inline tidb-cloud/ticloud-connect.md:27]: What is the default user? The root user?

File: tidb-cloud/ticloud-connect.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+```shell
+ticloud connect -p <project-id> -c <cluster-id>
+```
+
+Use a specific user to connect to the TiDB Cloud cluster in non-interactive mode:
```
Comment (qiancai) [inline tidb-cloud/ticloud-connect.md:33]: Do customers need to create the specific user in advance?

File: tidb-cloud/ticloud-connect.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+## Flags
+
+In non-interactive mode, you need to manually enter required flags. In interactive mode, you can just follow CLI prompts to fill in them.
+
+| Flag                    | Description                                      | Required | Note                                                 |
+|-------------------------|--------------------------------------------------|----------|------------------------------------------------------|
+| -c, --cluster-id string | Cluster ID                                       | Yes      | Only works in non-interactive mode.                  |
+| -h, --help              | Help information for this command                | No       | Works in both non-interactive and interactive modes. |
+| -p, --project-id string | Project ID                                       | Yes      | Only works in non-interactive mode.                  |
+| -u, --user string       | The user for login if not using the default user | No       | Only works in non-interactive mode.                  |
```
Comment (qiancai) [inline tidb-cloud/ticloud-connect.md:48]:
```suggestion
| -u, --user string       | A specific user for login | No       | Only works in non-interactive mode.                  |
```

File: tidb-cloud/ticloud-connect.md (hunk 4 above)
Comment (xuanyu66) [reply tidb-cloud/ticloud-connect.md:33]: Yes

File: tidb-cloud/ticloud-connect.md (hunk 3 above)
Comment (xuanyu66) [reply tidb-cloud/ticloud-connect.md:26]: For dedicated, it is root. For serverless, it is root with prefixes, such as "SfgdsWRdfs342.root"

Comment (xuanyu66) [reply tidb-cloud/ticloud-connect.md:26]: If they use CLI to create the cluster, there is a step to set the password for the root user.
If they want to reset the password, they need to go to the TiDB Cloud console.

File: tidb-cloud/ticloud-connect.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+Connect to a TiDB Cloud cluster:
+
+```shell
+ticloud connect [flags]
+```
+
+> **Note:**
+>
+> - If you are prompted about whether to use the default user, you can choose `Y` to use the default root user or choose `n` to specify another user. For [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters, the name of the default root user has a [prefix](/tidb-cloud/select-cluster-tier.md#user-name-prefix) such as `3pTAoNNegb47Uc8`.
+> - The connection forces the [ANSI SQL mode](https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html#sqlmode_ansi) mode for the session. To exit the session, enter `\q`.
```
Comment (hfxsd) [inline tidb-cloud/ticloud-connect.md:17]:
```suggestion
> - The connection forces the [ANSI SQL mode](https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html#sqlmode_ansi) for the session. To exit the session, enter `\q`.
```

File: tidb-cloud/ticloud-connect.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+Use a specific user to connect to the TiDB Cloud cluster in non-interactive mode:
+
+```shell
+ticloud connect -p <project-id> -c <cluster-id> -u <user-name>
+```
+
+## Flags
+
+In non-interactive mode, you need to manually enter required flags. In interactive mode, you can just follow CLI prompts to fill in them.
```
Comment (hfxsd) [inline tidb-cloud/ticloud-connect.md:41]:
```suggestion
In non-interactive mode, you need to manually enter the required flags. In interactive mode, you can just follow CLI prompts to fill in them.
```

File: tidb-cloud/ticloud-connect.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+| -p, --project-id string | Project ID                                       | Yes      | Only works in non-interactive mode.                  |
+| -u, --user string       | A specific user for login | No       | Only works in non-interactive mode.                  |
+
+## Inherited flags
+
+| Flag                 | Description                                                                                          | Required | Note                                                                                                                     |
+|----------------------|------------------------------------------------------------------------------------------------------|----------|--------------------------------------------------------------------------------------------------------------------------|
+| --no-color           | Disables color in output.                                                                            | No       | Only works in the non-interactive mode. In the interactive mode, disabling color might not work with some UI components. |
```
Comment (hfxsd) [inline tidb-cloud/ticloud-connect.md:54]:
```suggestion
| --no-color           | Disables color in output.                                                                            | No       | Only works in non-interactive mode. In interactive mode, disabling color might not work with some UI components. |
```

Comment (qiancai) [reply tidb-cloud/ticloud-connect.md:54]: @xuanyu66  Would you please also update other places for this sentence?

File: tidb-cloud/ticloud-connect.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+| -P, --profile string | Specifies the active [user profile](/tidb-cloud/cli-reference.md#user-profile) used in this command. | No       | Works in both non-interactive and interactive modes.                                                                     |
+
+## Feedback
+
+If you have any questions or suggestions on the TiDB Cloud CLI, feel free to create an [issue](https://github.com/tidbcloud/tidbcloud-cli/issues/new/choose). Also, we welcome any contributions.
```
[the remark this answers is not in the document]
Comment (xuanyu66) [reply tidb-cloud/ticloud-connect.md:59]: Later, I will write contribute guide, and link them

**Review by hfxsd — APPROVED** [review]

**Review by zhangyangyu — APPROVED** [review]

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 10]
```diff
@@ -8,6 +8,14 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## February 17, 2023
+
+**CLI changes**
+
+- Add new command [`ticloud connect`](/tidb-cloud/ticloud-connect.md)
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:15]:
```suggestion
- Add a new command [`ticloud connect`](/tidb-cloud/ticloud-connect.md) for [TiDB Cloud CLI](/tidb-cloud/cli-reference.md).
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 11]
```diff
@@ -8,6 +8,14 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    `ticloud connect` is a command that allows you to connect to a TiDB Cloud cluster from your local machine. You can use `ticloud connect` to connect to a TiDB Cloud cluster and execute SQL statements.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:17]:
```suggestion
    `ticloud connect` is a command that allows you to connect to your TiDB Cloud cluster from your local machine without installing any SQL clients. After connecting to your TiDB Cloud cluster, you can execute SQL statements in the TiDB Cloud CLI.
```

**Review by hfxsd — APPROVED** [review]

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 12]
```diff
@@ -8,6 +8,14 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## February 17, 2023
+
+**CLI changes**
+
+- Add a new command [`ticloud connect`](/tidb-cloud/ticloud-connect.md) for [TiDB Cloud CLI](/tidb-cloud/cli-reference.md).
+
+  `ticloud connect` is a command that allows you to connect to your TiDB Cloud cluster from your local machine without installing any SQL clients. After connecting to your TiDB Cloud cluster, you can execute SQL statements in the TiDB Cloud CLI.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:17]:
```suggestion
    `ticloud connect` is a command that allows you to connect to your TiDB Cloud cluster from your local machine without installing any SQL clients. After connecting to your TiDB Cloud cluster, you can execute SQL statements in the TiDB Cloud CLI.
```

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**